src/main/java/seedu/address/logic/commands/EditTaskCommand.java

```diff
@@ -0,0 +1,119 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ENDTIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_STARTTIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_TASKS;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.EditTaskDescriptor;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.name.Name;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.task.Date;
+import seedu.address.model.task.EndTime;
+import seedu.address.model.task.StartTime;
+import seedu.address.model.task.Task;
+
+/**
+ * Edits the details of an existing person in the address book.
+ */
+public class EditTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "edit-t";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the person identified "
+            + "by the index number used in the displayed person list. "
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_DATE + "DATE] "
+            + "[" + PREFIX_STARTTIME + "START TIME] "
+            + "[" + PREFIX_ENDTIME + "END TIME] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
```

Comment:
I think the [c/PERSON_NAME] description is missing

Reply:
I think the [c/PERSON_NAME].. description is also missing from the AddTaskCommand class. Do you might adding this in? Thanks!